apps/els_lsp/src/els_unused_includes_diagnostics.erl

```diff
@@ -52,15 +51,24 @@ source() ->
 %%==============================================================================
 %% Internal Functions
 %%==============================================================================
--spec find_unused_includes(els_dt_document:item(), [poi()]) -> [uri()].
-find_unused_includes(#{uri := Uri} = Document, Includes) ->
+-spec find_unused_includes(els_dt_document:item()) -> [uri()].
+find_unused_includes(#{uri := Uri} = Document) ->
   Graph = expand_includes(Document),
-  POIs = els_dt_document:pois(Document),
+  POIs = els_dt_document:pois(Document,
```

Comment:
I wonder if it's worth maintaining an explicit list of definitions here. Did you measure the gain in scanning only these POIs vs scanning all of them?